### PR TITLE
Update Spring Cloud Azure Support File

### PR DESCRIPTION
**Pull request by Netyyyy**
Update Spring Boot and Spring Cloud versions for the compatibility test.

If you merged this PR, please update [Spring Cloud Azure Timeline](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-Timeline).

This PR is created by GitHub Actions: https://github.com/Azure/spring-cloud-azure-tools/actions/runs/3317269850

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-supported-spring.json [hunk 1]
```diff
@@ -11,7 +11,15 @@
     "current" : false,
     "releaseStatus" : "PRERELEASE",
     "snapshot" : false,
-    "supportStatus" : "TODO",
+    "supportStatus" : null,
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-supported-spring.json:14]: Does `supportStatus` should be updated manually?

Comment (saragluna) [reply sdk/spring/spring-cloud-azure-supported-spring.json:14]: now it should be

**Review by moarychan — APPROVED** [review]
LGTM, please help to enable this function to feature/spring-boot-3 branch.

**Review by chenrujun — APPROVED** [review]

**Comment by chenrujun** [comment]
@Netyyyy could you please write a document about this json file?

It should include these items:
1. What is it used for?
2. What does each field mean?
3. How does it updated?

**Pull request merged**